config/locales/views/advice_pages/electricity_recent_changes.yml

```diff
@@ -18,6 +18,16 @@ en:
         summary: This section gives a more detailed analysis of how your school’s electricity consumption has changed recently. It can be particularly useful if trying to investigate a change in electricity consumption or monitoring the impact of action taken in the school.
         title: Analysis
       insights:
+        how_do_you_compare:
+          summary: Compare your recent electricity use against other schools in your group
+          title: How do you compare?
+        link: Learn more
         next_steps: Next steps
-        title: Insights
+        summary: This page makes note of whether your electricity use has increased or decreased recently. Monitoring recent changes to your energy use can help you to assess the impacts of changes you have made, or quickly identify problems
+        title: What do we mean by recent changes?
+        your_recent_electricity_use:
+          table:
+            date_range: Date range
```

Comment:
Call this "period" instead. And I think there's a key for that in `advice_pages.yml` already.